release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -71,6 +71,13 @@ With this release, you can now hibernate your {product-title} cluster for up to
 
 For more information, see xref:../backup_and_restore/hibernating-cluster.adoc#hibernating-cluster[Hibernating an {product-title} cluster].
 
+[id="ocp-4-18-etcd-backup-restore_{context}"]
+==== Enhanced etcd backup and restore documentation
+
+To aid for quicker recovery of the cluster in some cases where a full cluster restoration from a previous backup is not necessary, the etcd disaster recovery documentation was updated and simplified. In addition to the normal disaster recovery backup restore procedure, an additional procedure was added to more quickly recover the cluster when at least one good node exists. You can choose any of the surviving nodes to run the recovery.
```

Comment:
I feel like the first sentence here reads better if we reverse the clauses and remove "to aid", like so:

```adoc
The etcd disaster recovery documentation was updated and simplified for quicker recovery of the cluster in cases where a full cluster restoration from a previous backup is not necessary.
```